Fix paths.
PREFIX=/usr on Debian
--- a/src/events/privmsg.cpp
+++ b/src/events/privmsg.cpp
@@ -51,7 +51,7 @@
 #elif defined(UNIX) && USE_LIBNOTIFY
 #include <libnotify/notify.h>
 #define SUMMARY_TEXT "Swirc IRC client"
-#define SWIRC_ICON "/usr/local/share/swirc/swirc-royal.png"
+#define SWIRC_ICON "/usr/share/swirc/swirc-royal.png"
 #endif
 
 #define NICK_S1 Theme("nick_s1")
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -532,7 +532,7 @@
 
 #ifdef HAVE_LIBINTL_H
 #if defined(UNIX)
-	if (bindtextdomain("swirc", "/usr/local/share/locale") == NULL) {
+	if (bindtextdomain("swirc", "/usr/share/locale") == NULL) {
 		err_ret("bindtextdomain");
 		return EXIT_FAILURE;
 	}
